### DISCUSSION TITLE
State of this library

### DISCUSSION THREAD

**Discussion by sinedoOo** [spatie/laravel-prometheus#45] [Q&A]
Hello, is this library feature ready? I didn't find any useful details in your documentation how to use something beyond simple gauge. For example counters or histograms.

**Comment by Floppy012** [comment]
Counters do exist but are not persisted in any way (the counter value has to be set when registering it). But that's about it. Neither Histogram nor Summaries are supported.

**Reply by Husseinhewehii** [reply]
because your caching is not configured properly with prometheus

**Comment by scofield-ua** [comment]
In my case, I end up using this library:
https://github.com/PromPHP/prometheus_client_php
